homeassistant/components/econet/__init__.py

```diff
@@ -77,6 +80,13 @@ async def async_setup_entry(hass, config_entry):
 
     api.subscribe()
 
+    def update_published():
```

Comment:
Will this be called from inside the event loop or from another thread?

Reply:
This will be called when pyeconet gets a response on an mqtt topic from paho-mqtt so I guess it would be from another thread?

Reply:
Ok, then we need to use the sync api inside this method.